Content.Server/HeightAdjust/BloodstreamAffectedByMassComponent.cs

```diff
@@ -0,0 +1,26 @@
+using Content.Server.Body.Components;
+
+namespace Content.Server.HeightAdjust;
+
+/// <summary>
+///     When applied to a humanoid or any mob, adjusts their blood level based on the mass contest between them
+///     and an average humanoid. <br/><br/>
+///
+///     The formula for the resulting bloodstream volume is <code>V = BloodMaxVolume * MassContest^Power</code>
+///     clamped between the specified Min and Max values.
+/// </summary>
+[RegisterComponent]
+public sealed partial class BloodstreamAffectedByMassComponent : Component
```

Comment:
This could probably be renamed to "MassModifiersComponent", or some variety of such, so that it could be extended into a modular component that handles all MassContest related subscriptions for a variety of systems.

Reply:
That is the opposite of modular.

Reply:
Mmm. Alright.